core/Cargo.toml

```diff
@@ -23,7 +23,7 @@ std = ["miden-crypto/std", "math/std", "winter-utils/std"]
 [dependencies]
 math = { package = "winter-math", version = "0.7", default-features = false }
 # miden-crypto = { package = "miden-crypto", version = "0.8", default-features = false }
-miden-crypto = { package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto", branch = "next", default-features = false }
+miden-crypto = { package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto", branch = "plafer-smt-merkle-store", default-features = false }
```

Comment:
We should be able to merge https://github.com/0xPolygonMiden/crypto/pull/264 and roll back this change.

Reply:
Done